src/test/java/edu/harvard/iq/dataverse/mydata/MyDataUtilTest.java

```diff
@@ -0,0 +1,74 @@
+package edu.harvard.iq.dataverse.mydata;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeTrue;
+
+import org.junit.experimental.theories.DataPoints;
+import org.junit.experimental.theories.Theories;
+import org.junit.experimental.theories.Theory;
+import org.junit.runner.RunWith;
+
+@RunWith(Theories.class)
+public class MyDataUtilTest {
```

Comment:
I hadn't seen JUnit theories before. They look really powerful and I like how you've used them here. They are experimental though and it looks like JUnit 5 is steering away from this sort of functionality https://github.com/junit-team/junit5/pull/1422#issuecomment-389644868

I'm ok with accepting this test. Can you leave a comment in the code about this experimentally for us to refer to when we upgrade?

Reply:
hi @matthew-a-dunlap 

Thanks for the review. I added a comment about JUnit Theories in a new commit.

Reply:
Could this be replaced with the upstream-supported Dynamic Testing in JUnit5?

Promising blog article on that: https://blog.codefx.org/libraries/junit-5-dynamic-tests/